src/tools/singlejar/port.cc

```diff
@@ -0,0 +1,36 @@
+// Copyright 2018 The Bazel Authors. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifdef _WIN32
+
+#include "src/tools/singlejar/port.h"
+
+#ifndef WIN32_LEAN_AND_MEAN
```

Comment:
`WIN32_LEAN_AND_MEAN` is unnecessary: https://blogs.msdn.microsoft.com/oldnewthing/20091130-00/?p=15863

Reply:
The macro will speed up the compilation of this file because `windows.h` is huge without it. Most Windows projects will use this macro to improve build time: https://github.com/search?q=org%3Agoogle+WIN32_LEAN_AND_MEAN&type=Commits